package.json

```diff
@@ -3,8 +3,8 @@
   "name": "govuk-frontend-repository",
   "description": "Used only for the development of GOV.UK Frontend, see `package/package.json` for the published `package.json`",
   "engines": {
-    "node": "^16.13.0",
-    "npm": "^8.1.0"
+    "node": "^16.13.0 || ^18.12.0",
+    "npm": "^8.1.0 || ^9.1.0"
```

Comment:
I've included `npm@9` as it's being [backported to Node.js 18 in December](https://github.com/npm/statusboard/issues/587)

More info via [**npm CLI v9 is now generally available**](https://github.blog/changelog/2022-11-09-npm-cli-v9-is-now-generally-available/) from GitHub Changelog